mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -84,7 +85,9 @@ CLASS net/minecraft/class_57 net/minecraft/entity/Entity
 	METHOD method_1370 tick ()V
 	METHOD method_1371 move (DDD)V
 	METHOD method_1375 setLevel (Lnet/minecraft/class_18;)V
+		ARG 1 level
```

Comment:
not all parameters need to be mapped

Reply:
if ff automatically sets it the same as what you'll name it, don't set the parameter name

Reply:
pls